resume.html

```diff
@@ -0,0 +1,240 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"
+      integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="./css/main.css" />
+
+    <title>My website | Resume</title>
+  </head>
+
+  <body>
+    <div class=".container">
+      <!-- Header section -->
+      <header>
+        <nav class="nav second-nav">
+          <input
+            type="checkbox"
+            name=""
+            id="nav-checkbox"
+            class="nav-checkbox"
+          />
+          <label for="nav-checkbox" class="nav-toggle">
+            <img
+              class="menu"
+              src="./img/burger-menu.svg"
+              width="100"
+              alt="burger menu image"
+            />
+            <svg class="close" viewBox="0 0 384 512" width="100" title="times">
+              <path
+                d="M242.72 256l100.07-100.07c12.28-12.28 12.28-32.19 0-44.48l-22.24-22.24c-12.28-12.28-32.19-12.28-44.48 0L176 189.28 75.93 89.21c-12.28-12.28-32.19-12.28-44.48 0L9.21 111.45c-12.28 12.28-12.28 32.19 0 44.48L109.28 256 9.21 356.07c-12.28 12.28-12.28 32.19 0 44.48l22.24 22.24c12.28 12.28 32.2 12.28 44.48 0L176 322.72l100.07 100.07c12.28 12.28 32.2 12.28 44.48 0l22.24-22.24c12.28-12.28 12.28-32.19 0-44.48L242.72 256z"
+              />
+            </svg>
+          </label>
+          <ul class="nav-menu">
+            <li>
+              <a href="#"><div class="square"></div></a
+              ><strong>Maya Nelson</strong> <br /><span class="occupation"
+                >PROJECT MANAGER</span
+              >
+            </li>
+            <li><a href="index.html">ABOUT ME</a></li>
+            <li><a class="active" href="resume.html">RESUME</a></li>
+            <li><a href="projects.html">PROJECTS</a></li>
+            <li><a href="contact.html">CONTACT</a></li>
+          </ul>
+        </nav>
+      </header>
+
+      <!-- Main section - Resume -->
+      <main>
+        <!-- Resume -->
+        <section class="resume bolder">
```

Comment:
I'd add padding left and right same on this big container, and remove from small articles div to center elements.

before
<img width="907" alt="스크린샷 2023-03-06 00 15 14" src="https://user-images.githubusercontent.com/63498427/222994388-38908170-a776-4b80-b95c-257daac1f17a.png">

after
<img width="903" alt="스크린샷 2023-03-06 00 12 28" src="https://user-images.githubusercontent.com/63498427/222994378-7baaa41c-8981-4fc5-a955-f387983e783f.png">


CSS code for centering
1. add padding-right: 2rem to resume
<img width="250" alt="스크린샷 2023-03-06 00 15 43" src="https://user-images.githubusercontent.com/63498427/222994409-f93823d4-dd3b-423e-846b-432c34aa63d5.png">

2. remove width and set padding left and right same on article
<img width="254" alt="스크린샷 2023-03-06 00 12 43" src="https://user-images.githubusercontent.com/63498427/222994445-5a935fec-aa0a-4371-94c9-41dffe090947.png">